From wiget@usa.net  Thu Mar  4 22:01:41 1999
Return-Path: <wiget@usa.net>
Received: from ikar.t17.ds.pwr.wroc.pl (ikar.t17.ds.pwr.wroc.pl [::ffff:156.17.210.253])
	by wiget.t17.ds.pwr.wroc.pl (8.9.3/8.9.3) with ESMTP id WAA02847
	for <wiget@wiget.t17.ml.org>; Thu, 4 Mar 1999 22:01:37 +0100
Received: from mx04.netaddress.usa.net (mx04.netaddress.usa.net [204.68.24.141])
	by ikar.t17.ds.pwr.wroc.pl (8.8.7/8.8.8/nospam) with SMTP id VAA11615
	for <wiget@ikar.t17.ds.pwr.wroc.pl>; Thu, 4 Mar 1999 21:58:22 +0100
Received: (qmail 20733 invoked by uid 0); 4 Mar 1999 20:39:54 -0000
Received: from roadrunner.sig.net [192.195.85.203] by mx04 via mtad (2.6) 
	with ESMTP id 813DcDun20006M04; Thu, 04 Mar 1999 20:39:53 GMT
Received: (from majordom@localhost) by roadrunner.sig.net (8.9.0/8.8.6) id OAA26122; Thu, 4 Mar 1999 14:01:55 -0600 (CST)
Message-ID: <19990304120101.39137@shell3.ba.best.com>
Date: Thu, 4 Mar 1999 12:01:01 -0800
From: Kim DeVaughn <kimdv@best.com>
To: Lynx Developers <lynx-dev@sig.net>
Subject: Re: lynx-dev undefined reference to `LIsListpageTitle'
Mail-Followup-To: Lynx Developers <lynx-dev@sig.net>
References: <19990304191726.A27299@wiget.t17.ds.pwr.wroc.pl> <19990304112426.31147@shell3.ba.best.com>
Mime-Version: 1.0
Content-Type: text/plain; charset=us-ascii
X-Mailer: Mutt 0.88.14i
In-Reply-To: <19990304112426.31147@shell3.ba.best.com>; from Kim DeVaughn on Thu, Mar 04, 1999 at 11:24:26AM -0800
Organization: Kim's Home for Wayward Cocktail Waitresses & Hors d'oeuvre Girls
Sender: owner-lynx-dev@sig.net
Precedence: bulk
Reply-To: lynx-dev@sig.net
Status: RO
Content-Length: 891
Lines: 31

On Thu, Mar 04, 1999, I said:
|
| Looks like a typo at LYMainLoop.c:3221.  That should be LYIsListpageTitle,
| I do believe.
|
| To wit:

Sigh.

I somehow nuked the diff header lines of the patchlet to fix that.

Here it is again ... in a form that patch(1) will understand ...

/kim


Against 2.8.2dev.18:

diff -uNr lynx-2.8.2-dev.18+kd.orig/src/LYMainLoop.c lynx-2.8.2-dev.18+kd/src/LYMainLoop.c
--- lynx-2.8.2-dev.18+kd.orig/src/LYMainLoop.c	Thu Mar  4 02:39:45 1999
+++ lynx-2.8.2-dev.18+kd/src/LYMainLoop.c	Thu Mar  4 11:33:49 1999
@@ -3218,7 +3218,7 @@
 			 *  List Page is about. - kw
 			 */
 			if ( 0==strcmp(curdoc.address, LYlist_temp_url()) &&
-			    (LIsListpageTitle(curdoc.title ? curdoc.title : ""))) {
+			    (LYIsListpageTitle(curdoc.title ? curdoc.title : ""))) {
 			    if (!curdoc.post_data ||
 				/*
 				 *  Normal case - List Page is not associated
##--eof--##

